packages/odyssey-react/src/hooks/index.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import useFocusTrap from './useFocusTrap';
+import useKeypress from './useKeypress';
+import useOutsideClick from './useOutsideClick';
+
+export {
+  useFocusTrap,
+  useKeypress,
+  useOutsideClick
+}
```

Comment:
strong work! 💪 

let's move these into `src/utils`. No need to include `src/hooks` yet and we already have hooks in `utils`. Since we have about 100 new lines of JS logic behind these hooks having tests is going to be important too. `@testing-library/react-hooks` and `renderHook` will let us unit test these with an easier level of abstraction. We should try for some basic coverage here before we merge a PR with this code as part of it!

Reply:
Totally agree, will write the tests. Was just looking for some high-level validation on the approach. If we're good with it ill make sure the tests are in there!

Reply:
Nothing sticks out or raises any flags for me with these. Exactly the kind of reusable behavior that belongs in hooks.